### PR TITLE
WID-35. Setup SonarCloud analysis.

### PR DESCRIPTION
**Pull request by popaula937**

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,7 @@
+sonar.organization=the-virtual-brain
+sonar.projectKey=the-virtual-brain_tvb-ext-unicore
+
+sonar.python.version=3.8, 3.9
```
Comment (liadomide) [inline sonar-project.properties:4]: Could this work also on Python 3.10 ?

**Review by liadomide — APPROVED** [review]

**Comment by liadomide** [comment]
Should we exclude src/*.ts files from the coverage report for now ?
Or better leave the scan to fail on coverage, until we also have tests for the front-end ?

**Comment by popaula937** [comment]
> Should we exclude src/*.ts files from the coverage report for now ? Or better leave the scan to fail on coverage, until we also have tests for the front-end ?

I think it would be nice to have tests for the front-end, so I vote to let it fail until we add them. We should add a task in jira for that.

File: sonar-project.properties (hunk 1 above)
Comment (popaula937) [reply sonar-project.properties]: I added it and the sonar scan is passing, the result is the same. I am curious whether Sonar would indicate whether a code suggestion/vulnerability is part of a certain version of python or would just display it generically.

**Pull request merged**